scripts/monitor/processEvents.js

```diff
@@ -66,6 +67,105 @@ async function processNoParcelRelatedEvents(event) {
       }
       break
     }
+    case BlockchainEvent.EVENTS.startedMortgage: {
+      const { _id } = event.args
+      const block_time_updated_at = await new BlockTimestampService().getBlockTime(
+        block_number
+      )
+      try {
+        log.info(`[${name}] Starting Mortgage ${_id}`)
+        await Mortgage.update(
+          {
+            status: Mortgage.STATUS.ongoing,
+            block_time_updated_at
+          },
+          {
+            mortgage_id: _id
+          }
```

Comment:
_Opinion_: I think that `{ mortgage_id: _id }` looks a bit better, same with `loan_id: _index` and the others below. If you agree and don't mind changing it, I'm all for it.